CHANGES.md

```diff
@@ -17,22 +17,6 @@
   + Remove some calls to if_newline and break_unless_newline and fix break before closing brackets (#1168) (Guillaume Petiot)
   + Fix unstable cmt in or-pattern (#1173) (Guillaume Petiot)
 
-#### Internal
-
-  + Work with base v0.13.0 (#1163) (Jules Aguillon)
```

Comment:
Should we keep this one ? It impacts end-users but they can have that informations from opam

Reply:
Breaking changes can be in the Changes section

Reply:
Good point, I added a section for this.